iOS/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -177,6 +230,26 @@ extension CourseListViewController: UISearchControllerDelegate {
             statusBarBackground.autoresizingMask = [.flexibleWidth]
             self.statusBarBackground = statusBarBackground
         }
+
+        self.updateSearchFilterContainerHeight(isSearching: true)
+        self.collectionViewLayout.invalidateLayout()
+
+        // swiftlint:disable:next trailing_closure
+        UIView.animate(withDuration: 0.25, delay: 0, options: .curveEaseInOut, animations: {
+            self.collectionView.layoutIfNeeded()
+        })
+    }
+
+    func willDismissSearchController(_ searchController: UISearchController) {
+        self.updateSearchFilterContainerHeight(isSearching: false)
+        self.collectionViewLayout.invalidateLayout()
+
+        // swiftlint:disable:next trailing_closure
```

Comment:
Same as the 2 comments above.

Reply:
When using a trailing closure, swift compiler can't resolve the method 😞